### PR TITLE
Refactor/campaign storage

### PR DESCRIPTION
**Pull request by tcrosado**
This PR carries some **breaking changes** as information like Package Name, vercodes and Countries are not longer stored by Advertisement storage.

The country code conversion to integer is still present but it's no longer used in the contracts. Register PoA is also still present. 

Considering gas cost, this implementation reduces gas from 466716 gas to 203650 gas. This represents a 56% reduction on gas when creating a campaign.

### CODE REVIEW

File: test/advertisement.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ var expectRevert = RegExp('revert');
```
Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

**Review by GustavoRSSilva — APPROVED** [review]

**Comment by GustavoRSSilva** [comment]
**Reminder** wait for the API do develop the required resources before deploying.

File: test/advertisement.js (hunk 1 above)
Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

Comment (DioPires) [inline test/advertisement.js:21]: #### Tests:
```diff 
+ 32 passing
```
#### Test coverage: 
81.85 %

File: test/advertisement.js [hunk 2]
```diff
@@ -1,6 +1,5 @@
 var Advertisement = artifacts.require("./Advertisement.sol");
```
Comment (DioPires) [inline test/advertisement.js]: #### Tests:
```diff 
- 3 failing
```
```diff 
+ 46 passing
```
#### Test coverage: 
84.8 %

Comment (DioPires) [inline test/advertisement.js]: #### Tests:
```diff 
- 3 failing
```
```diff 
+ 46 passing
```
#### Test coverage: 
84.8 %

Comment (DioPires) [inline test/advertisement.js]: #### Tests:
```diff 
+ 49 passing
```
#### Test coverage: 
84.8 %

Comment (DioPires) [inline test/advertisement.js]: #### Tests:
```diff 
+ 49 passing
```
#### Test coverage: 
84.8 %

**Pull request merged**